packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1294,9 +1294,9 @@ abstract class RenderBox extends RenderObject {
       child.parentData = BoxParentData();
   }
 
-  Map<_IntrinsicDimensionsCacheEntry, double> _cachedIntrinsicDimensions;
+  Map<_IntrinsicDimensionsCacheEntry, double/*!*/> _cachedIntrinsicDimensions;
```

Comment:
Presumably, `_IntrinsicDimensionsCacheEntry` would also be `/*!*/` ?

Reply:
somehow it got that right